backend/src/auth/mod.rs

```diff
@@ -135,10 +152,34 @@ pub(crate) struct User {
 }
 
 /// Returns a representation of the optional username useful for logging.
-pub(crate) fn debug_log_username(session: &Option<User>) -> Cow<'static, str> {
+pub(crate) fn debug_log_username(session: &AuthContext) -> Cow<'static, str> {
+    use AuthContext::*;
     match session {
-        None => "none".into(),
-        Some(user) => format!("'{}'", user.username).into(),
+        Anonymous => "none".into(),
```

Comment:
Yip, copy&paste error.

Reply:
Good question. Can't really tell you I think. I see that it could be confusing. I saw `None` being defined in others' peoples code as well. So :shrug: Maybe let's go with anonymous, then it's not confusing at least and "anonymous" is descriptive as well.

Reply:
Let's keep `Anonymous` for now. I will fix the string.